akka-persistence-query/src/main/scala/akka/persistence/query/scaladsl/ReadJournal.scala

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (C) 2009-2015 Typesafe Inc. <http://www.typesafe.com>
+ */
+
+package akka.persistence.query.scaladsl
+
+import akka.persistence.query.{ Hint, Query }
+import akka.stream.scaladsl.Source
+
+/**
+ * API for reading persistent events and information derived
+ * from stored persistent events.
+ *
+ * The purpose of the API is not to enforce compatibility between different
+ * journal implementations, because the technical capabilities may be very different.
+ * The interface is very open so that different journals may implement specific queries.
+ *
+ * Usage:
+ * {{{
+ * val journal = PersistenceQuery(system).readJournalFor(queryPluginConfigPath)
+ * val events = journal.query(EventsByTag("mytag", 0L))
+ * }}}
+ *
+ * For Java API see [[akka.persistence.query.javadsl.ReadJournal]].
+ */
+abstract class ReadJournal { self ⇒
```

Comment:
Right, thanks.

Was a leftover from trying out integrating event-adapters: https://github.com/akka/akka/issues/18050 One of the ways to integrate them is to implement them on the ReadJournal directly here. I dropped it, since it felt a bit like mixing up concerns. More impl ideas in that ticket.